server/app/util/matrix_proxy.py

```diff
@@ -73,6 +73,8 @@ class MatrixProxy(_ArrayProxyBase):
         "numpy.ndarray": True,
         "scipy.sparse.csc.csc_matrix": True,
         "scipy.sparse.csr.csr_matrix": True,
+        "anndata._core.sparse_dataset.backed_csr_matrix": "MatrixProxy_anndata_h5py",
```

Comment:
These do not belong in this (general purpose) file.  All ScanPy/AnnData driver code is in the scanpy_engine module.